pkg/controller/jobs/pod/pod_controller.go

```diff
@@ -495,13 +493,7 @@ func SetupIndexes(ctx context.Context, indexer client.FieldIndexer) error {
 	return nil
 }
 
-func CanSupportIntegration(opts ...jobframework.Option) (bool, error) {
-	options := jobframework.ProcessOptions(opts...)
-
-	v := options.KubeServerVersion.GetServerVersion()
-	if v.String() == "" || v.LessThan(kubeversion.KubeVersion1_27) {
-		return false, fmt.Errorf("kubernetesVersion %q: %w", v.String(), errPodNoSupportKubeVersion)
-	}
+func CanSupportIntegration(_ ...jobframework.Option) (bool, error) {
```

Comment:
Can we remove this function? It seems to be optional.

Reply:
removed